web/concrete/src/Page/Page.php

```diff
@@ -2836,6 +2836,7 @@ public function add($pt, $data, $template = false)
             $pkgID = 0;
         }
 
+        $data['name'] = null;
```

Comment:
@aembler my sample code is just matching what I write in my day-to-day, I'm sure joe knows not to use php 5.4+ syntax. IMO we shouldn't be making any pull requests against the release/8.0.0 branch until the tests are passing.